ext/reflection/php_reflection.c

```diff
@@ -1790,6 +1790,20 @@ ZEND_METHOD(ReflectionFunctionAbstract, isUserDefined)
 }
 /* }}} */
 
+/* {{{ Returns whether this function is an anonymous closure or not */
+ZEND_METHOD(ReflectionFunction, isAnonymous)
+{
+	reflection_object *intern;
+	zend_function *fptr;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+	GET_REFLECTION_OBJECT_PTR(fptr);
+	RETURN_BOOL((fptr->common.fn_flags & ZEND_ACC_FAKE_CLOSURE) == 0);
```

Comment:
won't this also return true if the function is a callable rather than a Closure object? e.g. `'strlen'`

Reply:
```suggestion
	RETURN_BOOL((fptr->common.fn_flags & (ZEND_ACC_FAKE_CLOSURE | ZEND_ACC_CLOSURE) == ZEND_ACC_CLOSURE);
```
Otherwise it'll include non-closures as well

Reply:
test added